src/util/configHelpers.js

```diff
@@ -671,6 +787,53 @@ const restructureListingFields = hostedListingFields => {
   );
 };
 
+///////////////////////////////////////
+// Restructure hosted listing config //
+///////////////////////////////////////
+const restructureUserFields = hostedUserFields => {
+  return (
+    hostedUserFields?.map(userField => {
+      const {
+        key,
+        scope,
+        schemaType,
+        enumOptions,
+        label,
+        filterConfig = {},
+        showConfig = {},
+        saveConfig = {},
+        ...rest
+      } = userField;
+      const defaultLabel = label || key;
+      const enumOptionsMaybe = ['enum', 'multi-enum'].includes(schemaType) ? { enumOptions } : {};
+      const { required: isRequired, ...restSaveConfig } = saveConfig;
+
+      return key
+        ? {
+            key,
+            scope,
+            schemaType,
+            ...enumOptionsMaybe,
+            filterConfig: {
+              ...filterConfig,
+              label: filterConfig.label || defaultLabel,
+            },
```

Comment:
There's no filterConfig for user fields. We should not add it (and make sure the code elsewhere doesn't expect this.

Reply:
Roger that, removed!